### ISSUE TITLE
Any plans for cdn support?

### ISSUE THREAD

**Issue by raafirivero** [xwp/wp-dependency-minification#51]
Would love to get store minified scripts on AWS for max speed.

**Comment by shadyvb** [comment]
+1 for that.

**Comment by westonruter** [comment]
Yeah, good idea. I suppose all this would need is a way to change out the base URL for the output script tags to be able to point to a CDN, and then the CDN can do an origin-pull?

**Comment by shadyvb** [comment]
I was thinking uploading to S3 and serving URLs from there, and i guess OP meant that as well, if i'm not mistaken.

**Comment by westonruter** [comment]
Origin pull would be much easier to implement.

**Comment by szepeviktor** [comment]
Try [tiny-cdn](https://github.com/szepeviktor/tiny-cdn/blob/master/tiny-cdn.php) is it 130 sloc only.
No fancy stuff, only speed.
